### PR TITLE
add dotnet10 feeds

### PR DESCRIPTION
**Pull request by lewing**

### CODE REVIEW

**Review by akoeplinger — APPROVED** [review]
LGTM but I guess we should also add the net9 manifests?

**Comment by lewing** [comment]
> LGTM but I guess we should also add the net9 manifests?

We need to fake it until the sdk supports net10

File: eng/workloads/workloads.csproj [hunk 1]
```diff
@@ -62,6 +62,10 @@
                           Version="$(AssemblyVersion)"
                           Title=".NET WebAssembly Build Tools (Emscripten)"
                           Description="Build tools for WebAssembly ahead-of-time (AoT) compilation and native linking."/>
+      <ComponentResources Include="microsoft-net-sdk-emscripten-net9"
+                          Version="$(AssemblyVersion)"
+                          Title=".NET WebAssembly Build Tools for .NET9 (Emscripten)"
+                          Description="Build tools for WebAssembly ahead-of-time (AoT) compilation and native linking. on .NET9"/>
```
Comment (akoeplinger) [inline eng/workloads/workloads.csproj:68]:
```suggestion
                          Description="Build tools for WebAssembly ahead-of-time (AoT) compilation and native linking on .NET9"/>
```

**Comment by lewing** [comment]
There is more to improve here but lets start by getting a 10.0.100 version of the workload out

**Pull request merged**